tests/helpers/yadda.js

```diff
@@ -0,0 +1,3 @@
+// needed for dummy app to run
```

Comment:
In a normal installation, the default blue print copies the equivalent file into the consuming apps tests/helpers folder.  It needed to be a blueprint file because browserify does not process the test-support tree.  And since the default blueprint is not run on the addon dummy app, this file needs to be there so the dummy app will work.